src/dsp_tools/commands/excel2xml/excel2xml_cli.py

```diff
@@ -146,7 +146,7 @@ def _convert_rows_to_xml(
 
         # this is a property-row
         else:
-            assert check_notna(row["prop name"])
+            assert check_notna(row["prop name"])  # noqa: S101 (use of assert in production code)
```

Comment:
any particular reason this is this ignored here? should be easy enough to resolve with a `if not check_notna(): raise AssertionError("...)` or something along these lines, right?